tests/t125-vector.c

```diff
@@ -0,0 +1,45 @@
+/// @file
+/// Test summing of a pair of vectors
+/// \test Test y = alpha x + beta y
+#include <ceed.h>
+#include <math.h>
+
+int main(int argc, char **argv) {
+  Ceed              ceed;
+  CeedVector        x, y;
+  CeedInt           n;
```

Comment:
See main for a few similar changes to these tests:
```suggestion
  CeedInt           len = 10;
```